.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -18,6 +18,9 @@ the problem you're facing:
    specific commit?
  - ...
 
+Note: If you are unsure whether your issue is a bug in the Zephyr codebase or if
```

Comment:
wouldnt this appear in each bug now? :)

Reply:
It's likely indeed. Didn't though about it. Alternative is a section in documentation to which we can refer to, but not sure it would have the same effect.

Reply:
There are other sections that users need to remove or fill in, this would be no different

Reply:
Yeah I'd also imagine the use would remove it with the rest of the template text, if it does not happen maybe we can put an explicit note asking to remote the text but it should be quite intuitive.

Reply:
> wouldnt this appear in each bug now? :)

@nashif Yes, but we currently have the following text which will be included in every bug report unless the submitter manually removes it, and this is merely adding to that.

> A clear and concise description of what the bug is.
> Please also mention any information which could help others to understand
the problem you're facing:
>  - What target platform are you using?
>  - What have you tried to diagnose or workaround this issue?
>  - Is this a regression? If yes, have you been able to "git bisect" it to a
   specific commit?
>  - ...

Reply:
yes yes, but this one appears as a *Note* at the end, ie.. it is not part of any section. I personally do not have a problem with this, but maybe the template needs to be tweaked in a way to force removal of placeholde text, i.e. put in the top and part of the first section most people will read and remove, not the buttom of the template.

Reply:
What about:
```
**Describe the bug**
A clear and concise description of what the bug is.
(Note that Github Discussions are available to first verify that the issue is a genuine Zephyr bug
and not a consequence of Zephyr services misuse.)
```

We can even add a link:
```
**Describe the bug**
A clear and concise description of what the bug is.
(Note that [Github Discussions](https://github.com/zephyrproject-rtos/zephyr/discussions) are available to first verify that the issue is a genuine Zephyr bug and not a consequence of Zephyr services misuse.)
```

Reply:
@erwango can you update the text to your proposal?

Reply:
> What about:

yes, makes sense.

Reply:
Updated